web/assets/js/repository.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function() {
+    $('#repository-star').click(function() {
+        var link = $(this);
+        $.ajax(
+            $(this).attr('href'),
+            {
+                method: 'PUT',
+                dataType: 'json',
+                success: function(data) {
```

Comment:
`success` callback is deprecated, you should use the `.done()` promise

see: http://api.jquery.com/jquery.ajax/#jqXHR